client/packages/coldchain/src/common/utils.ts

```diff
@@ -8,3 +9,12 @@ export const parseBreachType = (
 
   return { temperature, type };
 };
+
+export const useFormatTemperature = (temperature: number) => {
+  const { currentLanguage: language } = useIntlUtils();
+  return new Intl.NumberFormat(language, {
+    style: 'unit',
+    unit: 'celsius',
+    unitDisplay: 'short',
+  }).format(temperature);
```

Comment:
given that all of the usages of this function round before passing in the number, I would do this:

```suggestion
  }).format(NumUtils.round(temperature, 2));
```